readthedocs/projects/views/public.py

```diff
@@ -105,36 +106,23 @@ def get_context_data(self, **kwargs):
 @never_cache
 def project_badge(request, project_slug):
     """Return a sweet badge for the project"""
+    BADGE_PATH = "projects/badges/%s.svg"
```

Comment:
linting says this should be lower case.

Reply:
I also believe this probably needs to have a MEDIA_URL prepended to it, for production.